### PR TITLE
v1beta1 support for task and taskrun logs command

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds v1beta1 support for task and taskrun
adds option for dynmaic client creation

pipeline and pipelinerun logs with dynamic version detect is not
supported in this PR, only the test are modefied to work with
current change.

Signed-off-by: Pradeep Kumar pradkuma@redhat.com

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/actions/watch/watch.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package watch
```
Comment (danielhelfand) [inline pkg/actions/watch/watch.go:1]: Add copyright information

File: pkg/cmd/pipelineresource/create.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ Create a PipelineResource defined by foo.yaml in namespace 'bar':
 	}
 	f.AddFlags(c)
 	c.Flags().StringVarP(&opts.from, "from", "f", "", "local or remote filename to use to create the pipeline resource")
-	c.Flags().MarkDeprecated("from", "from and -f will be removed in v1.0.0. Use kubectl create -f instead. Learn more here: https://github.com/tektoncd/cli/issues/816")
+	_ = c.Flags().MarkDeprecated("from", "from and -f will be removed in v1.0.0. Use kubectl create -f instead. Learn more here: https://github.com/tektoncd/cli/issues/816")
```
Comment (danielhelfand) [inline pkg/cmd/pipelineresource/create.go:100]: What's the reasoning for using `_ =`?

File: pkg/actions/watch/watch.go (hunk 1 above)
Comment (pradeepitm12) [reply pkg/actions/watch/watch.go:1]: Ahh, thanks I will add it.

File: pkg/cmd/pipelineresource/create.go (hunk 2 above)
Comment (pradeepitm12) [reply pkg/cmd/pipelineresource/create.go:100]: It gives a linting error, hence explicitly ignored it.

File: pkg/task/tasklastrun.go [hunk 3]
```diff
@@ -17,11 +17,17 @@ package task
...
+	"github.com/tektoncd/cli/pkg/cli"
+	trlist "github.com/tektoncd/cli/pkg/taskrun/list"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+//Todo
+// changing this LastRun to use dynamic clinet leads to multiple other change in start commands
```
Comment (danielhelfand) [inline pkg/task/tasklastrun.go:29]:
```suggestion
// changing this LastRun to use dynamic client leads to multiple other change in start commands
```

**Comment by danielhelfand** [comment]
Looks good to me. Could you just squash the last commit (`address review comments`)?

**Pull request merged**